### PR TITLE
Translated zh_TW

### PR DESCRIPTION
**Pull request by olivertzeng**

### CODE REVIEW

**Comment by cassidyjames** [comment]
Thanks @olivertzeng! However, the localization of the app description seems to be a bit… loaded. 😅 I wonder if there is a nicer way to localize it?

**Comment by olivertzeng** [comment]
@cassidyjames What do you mean 'loaded'

**Comment by cassidyjames** [comment]
@olivertzeng my Google Translate may be a bit off, but while I understand localizing instead of just directly translating, talking about X/Twitter, military in Taiwan, and mental retardation(?) doesn't really keep the spirit of the original text.

**Comment by olivertzeng** [comment]
> @olivertzeng my Google Translate may be a bit off, but while I understand localizing instead of just directly translating, talking about X/Twitter, military in Taiwan, and mental retardation(?) doesn't really keep the spirit of the original text.

Well it's just real 'localizing' I can change it if you want. I swear it's the only part I do jokes

**Comment by olivertzeng** [comment]
Well it doesn't effect the translation I guess

**Comment by olivertzeng** [comment]
> @olivertzeng my Google Translate may be a bit off, but while I understand localizing instead of just directly translating, talking about X/Twitter, military in Taiwan, and mental retardation(?) doesn't really keep the spirit of the original text.

Hey @cassidyjames so do I need to correct my text? Or just stay like that, because I really just 'localized' these questions to our styles(these questions are probably the most concerned/curious ones)

File: po/zh_TW.po [hunk 1]
```diff
@@ -18,336 +18,341 @@ msgstr ""
...
 
 #: data/metainfo.appdata.xml.in:167
 msgid "Cassidy James Blaede"
-msgstr ""
+msgstr "Cassidy James Blaede"
 
 #: src/FortuneLabel.vala:16
 msgid "It is certain."
-msgstr ""
+msgstr "很有可能"
```
Comment (cassidyjames) [inline po/zh_TW.po:241]: From a quick Google Translate, this says it's "Probably", but this should be a "Yes" answer.

Perhaps I should add translator comments to make these more clear. 😄

File: po/zh_TW.po [hunk 2]
```diff
@@ -18,336 +18,341 @@ msgstr ""
...
-msgstr ""
+msgstr "問問題，得答案"
 
 #: data/metainfo.appdata.xml.in:10
 msgid ""
 "Does he love you? Should you have pizza for dinner? Is there such thing as a "
 "stupid question? Discover the answers to these questions and more with "
 "Clairvoyant, the magic 8-ball inspired fortune teller."
 msgstr ""
+"他喜歡你嗎？台灣會被武統嗎？X 會換回 Twitter 標誌嗎？"
```
Comment (cassidyjames) [inline po/zh_TW.po:45]: Yeah I would request that you change this text, please. The original is supposed to be light-hearted and inconsequential and I don't think I want to have the text talking about specific people, companies, and countries.

File: po/zh_TW.po [hunk 3]
```diff
@@ -18,336 +18,341 @@ msgstr ""
...
 
 #: data/launcher.desktop.in:5
 msgid "Ask questions and get psychic answers"
-msgstr ""
+msgstr "問問題，得答案"
 
 #: data/launcher.desktop.in:11
 msgid "magic;eight;ball;8;crystal;🔮;medium;🎱;"
-msgstr ""
+msgstr ";魔法;八號;球;水晶;🔮;中等;🎱;"
```
Comment (cassidyjames) [inline po/zh_TW.po:33]: No leading semicolon

```suggestion
msgstr "魔法;八號;球;水晶;🔮;中等;🎱;"
```

**Review by cassidyjames — CHANGES_REQUESTED** [review]
Thanks for your patience, @olivertzeng! I left a handful of comments, but this looks mostly good. 😄

**Comment by olivertzeng** [comment]
> Thanks for your patience, @olivertzeng! I left a handful of comments, but this looks mostly good. 😄 

Thanks for your reply! The legend is back🛐🛐🛐

Do I need to change something for you?

**Comment by cassidyjames** [comment]
@olivertzeng yes, please see the inline comments in the GitHub thread

**Comment by olivertzeng** [comment]
Okay thanks!

File: po/zh_TW.po (hunk 1 above)
Comment (olivertzeng) [reply po/zh_TW.po:273]: Thanks! Check out oldtwitter's translation! They have very clear comments.

Comment (olivertzeng) [reply po/zh_TW.po:273]: https://github.com/dimdenGD/OldTwitter/pull/488

**Comment by olivertzeng** [comment]
Fixed

File: po/zh_TW.po (hunk 2 above)
Comment (cassidyjames) [reply po/zh_TW.po:49]: @olivertzeng could you revisit this line, please? I would like to keep it light-hearted and not mention specific people, companies, and countries.

File: po/zh_TW.po [hunk 4]
```diff
@@ -113,269 +116,287 @@ msgstr ""
...
 
 #: data/metainfo.appdata.xml.in:76
 msgid "German translations thanks to @FineFindus on GitHub"
-msgstr ""
+msgstr "感謝 FineFindus 在 GitHub 上翻譯德文"
 
 #: data/metainfo.appdata.xml.in:77
 msgid "Updated French translations thanks to @rene-coty on GitHub"
+msgstr "感謝 FineFindus 在 GitHub 上翻譯德文"
 msgstr ""
```
Comment (cassidyjames) [inline po/zh_TW.po:128]:
```suggestion
```

**Comment by cassidyjames** [comment]
@olivertzeng sorry about all the git-weirdness, it turns out having the automatic translation updates action is nice, but can also create conflicts. 😬 But you should be good to update anything in this branch, and I'll make sure it gets merged correctly.

**Comment by olivertzeng** [comment]
👍🏼

**Comment by olivertzeng** [comment]
Ive resolved all the issues you gave and fixed a lot of stuff @cassidyjames

**Review by olivertzeng — COMMENTED** [review]
line 49 has changed

File: po/zh_TW.po (hunk 2 above)
Comment (olivertzeng) [reply po/zh_TW.po]: Solved

File: po/zh_TW.po [hunk 5]
```diff
@@ -18,336 +18,341 @@ msgstr ""
...
+msgstr "翻譯和 Gnome 44"
 
 #: data/metainfo.appdata.xml.in:46
 msgid "German translations thanks to @FineFindus on GitHub"
-msgstr ""
+msgstr "感謝 FineFindus 在 GitHub 上翻譯德文"
 
 #: data/metainfo.appdata.xml.in:47
 msgid "Updated French translations thanks to @rene-coty on GitHub"
+msgstr "感謝 FineFindus 在 GitHub 上翻譯德文"
```
[the remark this answers is not in the document]
Comment (olivertzeng) [reply po/zh_TW.po]: Solved

**Comment by olivertzeng** [comment]
> line 49 has changed

Can you please merge my pr?

**Comment by olivertzeng** [comment]
Oh wait it is merged

**Comment by olivertzeng** [comment]
Is it @cassidyjames just want to check

**Comment by cassidyjames** [comment]
@olivertzeng I'm actively working on fixing the conflicts. :) I plan to merge it today and do a new release soon.

**Comment by olivertzeng** [comment]
> @olivertzeng I'm actively working on fixing the conflicts. :) I plan to merge it today and do a new release soon.

Great to hear!

**Review by cassidyjames — APPROVED** [review]

**Pull request merged**